src/Sarif.Multitool.Library/MergeCommand.cs

```diff
@@ -173,14 +175,17 @@ private async Task<bool> MergeSarifLogsAsync()
 
                                 if (!_ruleIdToRunsMap.TryGetValue(key, out Run splitRun))
                                 {
-                                    splitRun = _ruleIdToRunsMap[key] = new Run()
-                                    {
-                                        Tool = emptyRun.Tool,
-                                        Results = new List<Result>()
-                                    };
+                                    emptyRun.Results = new List<Result>();
+                                    splitRun = _ruleIdToRunsMap[key] = emptyRun;
```

Comment:
>emptyRun [](start = 71, length = 8)

This might be the right start but I think we're not comprehensively handling all that we should. What are we doing about rule indexes and artifact location indexes for example? When we add a result from one run *into* another, we should be fixing up those indices, expanding the target run with new rule/artifact data if it comes in, etc. Our baselining/result matching code does this and has some utilities for it. This is all to say: we probably have more work to do in this command (which previously simply moved entire run objects around, a very simple operation). Let's open an issue noting the open work and keep moving. :)